src/main/java/org/apache/mesos/hdfs/state/LiveState.java

```diff
@@ -85,24 +86,24 @@ public void updateTaskForStatus(Protos.TaskStatus status) {
       // set the status to initialized
       if (status.getMessage().equals(HDFSConstants.NAME_NODE_INIT_MESSAGE)
           || (currentAcquisitionPhase.equals(AcquisitionPhase.RECONCILING_TASKS)
-              && !isNameNode1Initialized())
+          && !isNameNode1Initialized())
```

Comment:
Not sure I agree with this change in indentation, since this && is at a different () nesting level from the ||s